File: practicals/array/array-basics/p4.c
#include<stdio.h>
void main(){

        int N,product=1;
        printf("Enter NO of elements :");
        scanf("%d",&N);

        int arr[N];

        printf("Enter the values of elements: \n");

        for(int i=0;i<N;i++){

                scanf("%d",&arr[i]);
        }

        printf("\n");

        for(int i=0;i<N;i++){

                if(arr[i]%2==0){

                        product=product*arr[i];
                }
        }
        printf("Product of Even elements : %d",product);
        printf("\n--------------------------------------------------------------\n");
}

File: OOPS/java/p4.java

import java.util.Scanner;

public class main{

	public static void main(String[] args){

		Scanner sc = new Scanner(System.in);

		System.out.println("Enter the size of array:");

		int n = sc.nextInt();

		int[] arr=new int[n];

		System.out.println("Enter the elements of array: ");

		for(int i=0;i<n;i++){

			arr[i]=sc.nextInt();
		}

		int temp=arr[0];

		for(int i=0;i<n;i++){

			if(arr[i]>temp){

				temp=arr[i];
			}
		}

		System.out.println("Largest element in array is "+temp);
		sc.close();
	}
}

